jobsupervisor/job_supervisor_interface.go

```diff
@@ -20,7 +21,7 @@ type JobSupervisor interface {
 	Unmonitor() error
 
 	Status() string
-
+	ServiceStatus() []boshmonit.Service
```

Comment:
let's be consistent in naming. may be introduce a new structure:

```
type ProcessState struct {
   Name string
   State string
   Details map[string]string
}
```

and then have JobSupervisor include `ProcessStates`:

```
type JobSupervisor interface {
  ...
  ProcessStates() ([]ProcessState, error)
  ...
}
```